flexmeasures/cli/jobs.py

```diff
@@ -59,7 +60,11 @@ def run_worker(queue: str, name: Optional[str]):
         q_list,
         connection=connection,
         name=used_name,
-        exception_handlers=[handle_forecasting_exception],
+        exception_handlers=[
+            handle_forecasting_exception
+            if queue == "forecasting"
+            else handle_scheduling_exception
```

Comment:
I prefer raising a `NotImplementedError` here in case `queue not in ("forecasting", "scheduling")`.

Alternatively, we make `handle_scheduling_exception` more generic. It currently just stores the exception as job meta data (which seems generic already) and prints out that it was a "scheduling worker" that raised the exception (which we can make generic by using `queue_name = job.origin` inside the exception handler).

Reply:
I wanted to make RQ error handling more generic, but I saw the forecasting error handler already implementing some specific ideas. It's good to keep them separated.

I can raise a `NotImplementedError` regardless of the choice of error handler.

Reply:
`parse_queue_list` already throws an error if an unknown queue name is passed in.

Reply:
No, I agree to keep the forecasting error handler separate. I only object to using a custom handler for the `else` logic. That should use a generic handler instead.

Reply:
> `parse_queue_list` already throws an error if an unknown queue name is passed in.

That's already great. But when we add a new queue in the future, I'd like either to be informed by the lack of a custom error handler for the new queue, or have it automatically use our custom handler, which then prints out the name of the new queue correctly.